lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -287,6 +287,7 @@
     <datatype extension="xquest.xml" type="galaxy.datatypes.proteomics:XquestXML" mimetype="application/xml" display_in_upload="true"/>
     <datatype extension="spec.xml" type="galaxy.datatypes.proteomics:XquestSpecXML" mimetype="application/xml" display_in_upload="true"/>
     <datatype extension="qcml" type="galaxy.datatypes.proteomics:QCML" mimetype="application/xml" display_in_upload="true"/>
+    <datatype extension="mzqc" type="galaxy.datatypes.text:Json" subclass="true" display_in_upload="true" description="Quality control data in JSON format (https://github.com/HUPO-PSI/mzQC)."/>
```

Comment:
MassSpec quality control data in JSON ...?

Reply:
Sounds good. Then we should also fix this for `qcml` where the desciption has been added later (I copy pasted from this) https://github.com/galaxyproject/galaxy/blob/bbab98ab46b06b9216c07c5433c7fc1edc92f3a7/lib/galaxy/config/sample/datatypes_conf.xml.sample#L310

Will add a PR for dev.